addons/website_event_track/controllers/event_track.py

```diff
@@ -30,7 +30,12 @@ def _get_event_tracks_base_domain(self, event):
         if not request.env.user.has_group('event.group_event_user'):
             search_domain_base = expression.AND([
                 search_domain_base,
-                ['|', ('is_published', '=', True), ('is_accepted', '=', True)]
+                [('stage_status', 'in', ['published', 'announced']), ('is_published', '=', True)]
```

Comment:
I think the goal of the "announced" stage/status is to allow tracks that are not published (is_published = False) to be displayed in the tracks list and agenda (but not clickable).

With your new approach, these tracks will not be displayed anymore.

Reply:
The specification does not describe what would be the purpose of the "publishing switch" with the new semantic. Here, I made the assumption that the track stage defines the visibility and the accessibility of the tracks and the "publishing switch" can be used to hide tracks that are supposed to be visible (i.e: announced and published track).

Here, you want the "publishing switch" to control the accessibility of the track having the "announced" status ?

Reply:
Well all I see in the task specification is:
> => replace the booleans by a selection field

Which intends to make it easier to understand from a user point of view.
Here, you also change the functional behavior of announced/unpublished tracks, and I don't know if it's desired.

Maybe something you should discuss with the PO?